### ISSUE TITLE
Error reported when no success url was provided into form config (angular)

### ISSUE THREAD

**Issue by seyko1** [lyra/embedded-form-glue#301]
Hi,

I'm working with KRGlue and Angular 15,

When I use `KR.setFormConfig()` without providing `kr-get-url-success` property into `KRConfig` object,
web console display an error after submitting a successful payment.

![image](https://github.com/lyra/embedded-form-glue/assets/26870879/c7866a82-4f25-445d-84eb-281b328e35dc)

Although this parameter was not provided in github example, is it required ?

Is it a bad practice to manage form and payment answer in a single component?

My implementation : 

```js
// ngAfterViewInit()
const formToken = (await firstValueFrom(this.myService.getFormToken())).formToken;

KRGlue.loadLibrary(environment.lyraUrl, environment.publicKey)
  .then(({ KR }) =>
      KR.setFormConfig({
        formToken: formToken,
        "kr-language": "fr-FR",
        // "kr-get-url-success": "/success", 
      })
    )
    .then(({ KR }) => KR.onSubmit(this.onSubmit))
    .then(({ KR }) => KR.attachForm("#paymentForm"))
    .then(({ KR, result }) => {
      KR.showForm(result.formId)
    })
    .catch(error => this.message = `${error.message} (see console for more details)`);

// onSubmit callback
private onSubmit = (paymentData: KRPaymentResponse) => {
  this.myService.validate(paymentData.post())
    .subscribe({
      next: () => {
       // hash was verified with 200
        this.message = "Payment was successful.";
       // do other stuff..
      },
      error: () => {
        this.message = "Payment hash verification failed.";
      }
    });
}
```

**Comment by leunam217** [comment]
Hi @seyko1 , thank you for reporting the problem.

This seems to be an angular error. An angular library wraps  promise rejections making await+ try catch not working. (Changing our code to do .catch() directly solves the problem)
https://github.com/angular/angular/issues/31680

To solve this issue it's possible to disable promise rejection wrapping.
 
You can "Disable wrapping uncaught promise rejection" by setting DISABLE_WRAPPING_UNCAUGHT_PROMISE_REJECTION. 

Follow the [Angular guide](https://angular.io/guide/zone#setting-up-zonejs) and see the [configuration](https://github.com/angular/angular/blob/main/packages/zone.js/lib/zone.configurations.api.ts).

In short:

Create a zone-flags.ts file with:
`
 (window as any)._zone_symbol_DISABLE_WRAPPING_UNCAUGHT_PROMISE_REJECTION = true; 
 `
  Insert the following in polyfills.ts before import 'zone.js';:
`
 import './zone-flags';
 `
This might work best if you only throw Error objects and not strings.

source: last answer' of https://stackoverflow.com/questions/51951471/angular-custom-error-handler-not-getting-error-type-from-promise.

This solution has solved one issue we had. Feel free to say if it worked for you. 
In the future we may add a fix in our side.

I think you are following our example. Therefore I think the code you shared is not bad practice.
